skbio/metadata/_repr.py

```diff
@@ -126,6 +127,16 @@ def _format_positional_metadata_column(self, key, dtype):
         dtype_fmt = '<dtype: %s>' % str(dtype)
         return self._wrap_text_with_indent(dtype_fmt, key_fmt, 1)
 
+    def _process_interval_metadata(self):
+        if (hasattr(self._obj, "has_interval_metadata") and
```

Comment:
yes. For the classes that haven't been modified to adapt interval metadata, eg `TabularMSA`, some of their tests would fail if this is not checked.